scripts/install_bitcoind.sh

```diff
@@ -2,17 +2,9 @@
 
 set -ev
 
-export BITCOIND_VERSION=0.20.0
-
-if sudo cp ~/bitcoin/bitcoin-$BITCOIND_VERSION/bin/bitcoind /usr/local/bin/bitcoind
-then
-        echo "found cached bitcoind"
-else
-        mkdir -p ~/bitcoin && \
-        pushd ~/bitcoin && \
-        wget https://bitcoin.org/bin/bitcoin-core-$BITCOIND_VERSION/bitcoin-$BITCOIND_VERSION-x86_64-linux-gnu.tar.gz && \
-        tar xvfz bitcoin-$BITCOIND_VERSION-x86_64-linux-gnu.tar.gz && \
-        sudo cp ./bitcoin-$BITCOIND_VERSION/bin/bitcoind /usr/local/bin/bitcoind && \
-        popd
-fi
+BITCOIND_VERSION=${BITCOIN_VERSION:-0.20.1}
 
+docker pull ruimarinho/bitcoin-core:$BITCOIND_VERSION
+CONTAINER_ID=$(docker create ruimarinho/bitcoin-core:$BITCOIND_VERSION)
```

Comment:
In the future we could have our own `bitcoind` image pushed too.

Reply:
Yes, that's what I wanted to do for a while. Base images for our integration tests that contain `bitcoind`, `btcd` and `lnd`. That would speed up things considerably.

Reply:
Completely agree, this is a good temporary state though.